File: eval/fixtures/legacy-directory-backed-webforms-commerce/Billing/BillingRefundService.cs
namespace Acme.Legacy.Billing;

public class BillingRefundService
{
  private readonly BillingRefundRepository _billingRefundRepository = new BillingRefundRepository();

  public string RebuildRefundAudit(string refundId)
  {
    return _billingRefundRepository.WriteRefundAuditEntry(refundId);
  }
}

File: eval/fixtures/legacy-directory-backed-billing/src/services/billing-refund-service.js
const BillingRefundRepository = require('../repositories/billing-refund-repository');

class BillingRefundService {
  applySettlement(refundId) {
    const repository = new BillingRefundRepository();
    return repository.recordSettlement(refundId);
  }

  auditSettlement(refundId) {
    const repository = new BillingRefundRepository();
    return repository.loadSettlementAudit(refundId);
  }
}

module.exports = BillingRefundService;
